metadata/pyAlgoDDO.json

```diff
@@ -0,0 +1,65 @@
+{
+  "@context": ["https://w3id.org/did/v1"],
+  "id": "",
+  "nftAddress": "",
+  "version": "4.1.0",
+  "chainId": 8996,
+  "metadata": {
+    "created": "2023-08-01T17:09:39Z",
+    "updated": "2023-08-01T17:09:39Z",
+    "type": "algorithm",
+    "name": "CLi Algo",
+    "description": "Cli algo",
+    "author": "OPF",
+    "license": "https://market.oceanprotocol.com/terms",
+    "additionalInformation": {
+      "termsAndConditions": "True"
+    },
+    "algorithm": {
+      "language": "python",
+      "format": "docker-image",
+      "version": "0.1",
+      "container": {
+        "entrypoint": "python $ALGO",
+        "image": "oceanprotocol/algo_dockers",
+        "tag": "python-branin",
+        "checksum": "sha256:8221d20c1c16491d7d56b9657ea09082c0ee4a8ab1a6621fa720da58b09580e4"
+      }
+    }
+  },
+  "services": [
+    {
+      "id": "db164c1b981e4d2974e90e61bda121512e6909c1035c908d68933ae4cfaba6b0",
+      "type": "access",
+      "files": {
+        "datatokenAddress": "0x0",
+        "nftAddress": "0x0",
+        "files": [
+          {
+            "type": "url",
+            "url": "https://raw.githubusercontent.com/oceanprotocol/c2d-examples/main/branin_and_gpr/gpr.py",
+            "method": "GET"
+          }
+        ]
+      },
+      "timeout": 86400,
+      "serviceEndpoint": "http://10.84.128.6:8001"
```

Comment:
Are you sure this is the inet IP from c2d cluster?

Reply:
I'll remove this to avoid confusion, it's not used for anything. I've been using `metadata/freeComputeAlgoJS.json` and `metadata/freeComputeAlgoPY.json` for testing.

Reply:
ok, perfect